mods/vote_time.php

```diff
@@ -87,6 +122,65 @@
+        // AND target channel was set in config AND Raid was not shared to target channel before
+        if($count_att == 0 && $config->SHARE_AFTER_ATTENDANCE && !empty($config->SHARE_CHATS_AFTER_ATTENDANCE) && $perform_share){
+            // Share Raid to another Channel
+            $chat = $config->SHARE_CHATS_AFTER_ATTENDANCE;
+            // get all raid info - very detailed, because gyms->gym_id (ingress portal id) is different to raids->gym_id (the real one)
+            $request_gym = my_query(
+                "
+                SELECT 
+                    r.id AS id,
+                    r.user_id AS user_id,
+                    r.pokemon AS pokemon,
+                    r.pokemon_form AS pokemon_form,
+                    r.first_seen AS first_seen,
+                    r.start_time AS start_time,
+                    r.end_time AS end_time,
+                    r.gym_team AS gym_team,
+                    r.gym_id AS gym_id,
+                    r.move1 AS move1,
+                    r.move2 AS move2,
+                    r.gender AS gender,
+                    g.lon AS lon,
+                    g.lat AS lat,
+                    g.address AS address,
+                    g.gym_name AS gym_name,
+                    g.ex_gym AS ex_gym,
+                    g.show_gym AS show_gym,
+                    g.gym_note AS gym_note,
+                    g.gym_id AS gym_id2,
+                    g.img_url AS img_url           
+                FROM raids as r 
+                left join gyms as g on r.gym_id = g.id 
+                WHERE r.id = {$data['id']}
+                "
+            );
+            $answer_gym = $request_gym->fetch();
+            // Set text.
+            $text = show_raid_poll($answer_gym);
+            // Set keys.
+            $keys = keys_vote($answer_gym);
+            // Set reply to.
+            $reply_to = $chat;
+            // Send the message.
+            if($config->RAID_PICTURE) {
+                require_once(LOGIC_PATH . '/raid_picture.php');
+                $picture_url = raid_picture_url($data);
+                $tg_json[] = send_photo($chat, $picture_url, $text['short'], $keys, ['reply_to_message_id' => $reply_to, 'reply_markup' => ['selective' => true, 'one_time_keyboard' => true], 'disable_web_page_preview' => 'true'], true);
+            } else {
+                $tg_json[] = send_message($chat, $text['full'], $keys, ['reply_to_message_id' => $reply_to, 'reply_markup' => ['selective' => true, 'one_time_keyboard' => true], 'disable_web_page_preview' => 'true'], true);
+            }
+            // Telegram multicurl request.
+            curl_json_multi_request($tg_json);
+            // Update new entry
+            /*if ($config->RAID_PICTURE){
+                send_response_vote($update, $answer_gym,false,false); 
+             } else {
+                send_response_vote($update, $answer_gym); 
+             }*/
```

Comment:
Can this be removed or is it needed?
Was it for testing something or does it need to be un-comment?

Reply:
was coding/testing stuff before I had get_all_active_raid_channels in send_response_vote.php 
its deleted